packages/yoastseo/spec/scoring/assessors/collectionPages/seoAssessorSpec.js

```diff
@@ -1,7 +1,7 @@
-import EnglishResearcher from "../../../src/languageProcessing/languages/en/Researcher";
-import Assessor from "../../../src/scoring/collectionPages/seoAssessor.js";
-import Paper from "../../../src/values/Paper.js";
-import { checkAssessmentAvailability, checkUrls } from "../../specHelpers/scoring/seoAssessorTests";
+import EnglishResearcher from "../../../../src/languageProcessing/languages/en/Researcher";
+import Assessor from "../../../../src/scoring/assessors/collectionPages/seoAssessor.js";
+import Paper from "../../../../src/values/Paper.js";
```

Comment:
In a few of these files, the import end with `.js`, in others it's without `.js`. Let's settle on one way for all of these (I guess without `.js`).

Reply:
I found [this info on Stackoverflow](https://stackoverflow.com/questions/72491392/when-do-we-need-to-add-file-extension-when-importing-javascript-modules) where the way I understand it is that using file extension is actually better? because tools like Webpack can resolve it immediately. What do you think?

Reply:
OK, let's use the extension 👍